portal/server/api.js

```diff
@@ -871,17 +871,23 @@ const getDueReportDatesByBank = async (token, bankId) => {
   return response.data;
 };
 
+const downloadUtilisationReport = async (userToken, bankId, _id) => {
```

Comment:
while it's already validated earlier and so is useless I'm semi thinking we add bankId validation here as it's in every function in this file as an SSRF fail safe. Reckon that's overkill or helpful for someone reading this in the future thinking it's missing?

Reply:
Yeah, I didn't add as it's already in the route handler. For that reason I probably wouldn't duplicate here, but I can see the argument for being safe too, so happy to go with your call on this. Let me know.

Reply:
Hmm, I think leave as it is, validation at the router level is the better approach anyway so I guess we should move towards that!